src/App.js

```diff
@@ -18,6 +20,11 @@ function App() {
     fetchData();
   }, []);
 
+  const searchHandler = async () => {
+    const data = await fetchBusinesses(zip, search);
+
+    return setBusinesses(data);
```

Comment:
Here we will want to handle if a zip code is not entered, as it appears to create an error if not entered.